### DISCUSSION TITLE
Kurviger Optionen von Streckenverlauf

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#102] [Q&A] [answered]
Warum gibt es die Option kurvige oder extra kurvige Route nicht mehr 
Mit dieser Option wurde eine super Route gemacht und das ohne setzen von zusätzlichen Wegpunkten. Auch wurden Städte immer super umfahren und führten nicht durchs Stadtzentrum 🙂
Das funktionierte in Kurviger 2 pro hervorragend.

**Comment by ToToGSARider** [comment, marked as answer]
Das ist glaube ich eine rechtliche Geschichte. Cruiser ist ja nicht kurviger. Du kannst aber auf kurviger planen, so mache ich es, und importierst es dann in den Cruiser. Neuberechnung aus und los geht's.

 I think that's a legal story. Cruiser is not curvier. But you can plan on curvier, that's how I do it, and then import it into cruiser. Recalculation off and off you go.

**Reply by 1maxi2** [reply]
O.k. Danke So mache ich es momentan auch. Aber irgendwann möchte ich nur mehr eine einzige App verwenden. Und devemux hat ja kurviger entwickelt oder ?
Somit dürfte es ja kein Problem sein diese Option in Cruiser zu übernehmen

**Reply by ToToGSARider** [reply]
Das kann ich Dir nicht sagen. Ich bin 2020 zu kurviger gekommen in der Version 1 weil das WunderlinQ an meiner GS unterstützt wurde. Ich weiss nicht wie Robin und Emux zueinander gefunden haben. Robin hat kurviger erfunden soviel ich weiß. Ich bin mit der Lösung mit meiner Magenta Cloud dazwischen echt zufrieden. Ich kann die Daten Offline in der Cloud laden und die Karten Offline in der App und das ist genau mein Anspruch. 

I can not tell you that. I came to curvier in 2020 in version 1 because the WunderlinQ was supported on my GS. I don't know how Robin and Emux found each other. Robin invented kurviger as far as I know. I am really happy with the solution with my Magenta Cloud in between. I can load the data offline in the cloud and the maps offline in the app and that is exactly my requirement.

**Reply by devemux86** [reply]
If you check Google Play, you will see that Cruiser has been available on Google Play since 2013.

K1 / K2 came much later and used the Cruiser libraries.
Kurviger / Calimoto routing cannot be used in external applications.

---

Wenn Sie Google Play überprüfen, werden Sie sehen, dass Cruiser seit 2013 auf Google Play verfügbar ist.

K1 / K2 kamen viel später und verwendeten die Cruiser-Bibliotheken.
Das Kurviger / Calimoto Routing kann nicht in externen Anwendungen verwendet werden.

**Comment by DolomitesTom** [comment]
Als langjähriger K1 und K2 User war ich auch erst mal geschockt über die Trennung der Zusammenarbeit. Cruiser ist genial und alles funktioniert offline (bei 2 von mehreren Routing-Optionen): Planung, POI und Favoriten-Suche und Navigation. Wenn man sich darauf einlässt, braucht man keine kurvigen Routen mehr, was sowieso nur online funktioniert hat. Und der Fallback zwischen Kurviger und Brouter hat bei K1 und K2 nicht so gut funktioniert. Wer viel reist, erkennt schnell, dass es erschreckend viele Regionen gibt ohne oder nur mit schlechter Netzabdeckung. Da ist Cruiser einfach absolut verlässlich. Wenn man die kurvig geplanten Routen beibehalten will, muss man nur einige Shapingpunkte setzen, etwas aufwändiger bei der Planung, aber machbar. Die Weiterentwicklungen in Cruiser seit Ende von K2 sind phänomenal, die individuelle Anpassbarkeit von Cruiser konkurrenzlos. Klar, man muss sich damit beschäftigen, aber das macht man für sein Hobby doch gern, oder 😉?
Die Bezahlversion von Kurviger Tourer Web kann man ja parallel nutzen. Das passt immer noch gut zusammen.

**Reply by ToToGSARider** [reply]
Da kann ich jetzt nur zustimmen @DolomitesTom ich plane alle Touren auf kurviger.de vor und lade mir dann alle Offline Karten für den Urlaub auf das Handy und die Routen in die Cloud und los geht's.

**Reply by 1maxi2** [reply]
Gerade das ist für mich der große Vorteil von kurviger. Sitze unterwegs in einer Unterkunft. Höre mir Tipps von Einheimischen an und plane dann die Touren spontan meist nur für den nächsten Tag direkt am Handy.
